Dockerfile.dev

```diff
@@ -28,5 +28,10 @@ RUN scl enable rh-nodejs4 "npm install -g nodemon" && \
     mv conf-docker.json config/conf.json && \
     chmod -R ug+rw ./
 
+USER root
```

Comment:
I've added this block in here to match the main Dockerfile. This Dockerfile is currently broken for other reasons though (since at least early September I guess?), so I'm not sure if it's even worthwhile to keep this file at all?

Reply:
I am not up to date with that file either, but if it's going to be there, it's better to keep them as close as possible +1